src/Application/GameLuaBindings.cpp

```diff
@@ -90,18 +125,64 @@ void GameLuaBindings::init(lua_State *L) {
             SetCurrentMenuID(MenuType(screenIndex));
         }
     );
+}
+
+void _registerAudioBindings(sol::state_view &luaState, sol::table &table) {
+    table["audio"] = luaState.create_table_with(
+        "playSound", [](SoundId soundId, SoundPlaybackMode mode) {
+            pAudioPlayer->playSound(soundId, mode);
+        },
+        "playMusic", [](MusicId musicId) {
+            pAudioPlayer->MusicPlayTrack(musicId);
+        }
+    );
+}
+
+void _registerItemBindings(sol::state_view &luaState, sol::table &table) {
+    typedef std::function<bool(ItemId)> FilteItemFunction;
+
+    auto createItemTable = [&luaState](const ItemDesc& itemDesc) {
+        return luaState.create_table_with(
+            "name", itemDesc.name,
+            "level", itemDesc.uItemID_Rep_St
```

Comment:
I'm not entirely sure if this variable can be translated to "level" when we expose it to lua. Let me know if there's a better name to identify it.

Reply:
IMO this should do for now. We'll need to do an iteration to sync up lua & cpp names once we have sane cpp names.